src/ApiStatus.ts

```diff
@@ -91,11 +90,23 @@ export default class ApiStatus {
         message.channel.send({embed: embedContent});
     }
 
+    private getLastUpdate(): string {
+        const timeDiff = Date.now() - this.lastCheckTime;
+        const min = Math.floor(timeDiff / 1000 / 60);
+        if (min >= 1) {
+            const minutes = min == 1 ? "minute" : "minutes";
+            return "Last refresh: " + min + " " + minutes +" ago";
+        } else {
+            const sec = Math.floor((timeDiff - min * 60000) / 1000);
+            const seconds = sec == 1 ? "second" : "seconds";
+            return "Last refresh: " + sec + " " + seconds + " ago";
+        }
+    }
```

Comment:
We already depend on [pretty-ms](https://github.com/sindresorhus/pretty-ms) for the uptime command, so this can probably be simplified to `prettyMs(Date.now() - this.lastCheckTime, { verbose: true }) + " ago"` (with the appropriate import of course).